website/docs/concepts/html/events.mdx

```diff
@@ -141,7 +141,7 @@ fn MyComponent() -> Html {
             // Events can bubble so this listener might catch events from child
             // elements which are not of type HtmlInputElement
             //highlight-next-line
-            let input = target.and_then(|t| t.dyn_into::<HtmlInputElement>().ok());
+            let input: Option<HtmlInputElement> = target.and_then(|t| t.dyn_into::<HtmlInputElement>().ok());
```

Comment:
I believe the original would also work, the type can be inferred with the blowfish syntax.